### PR TITLE
Rule: SC2-5-3-label-content-name-mismatch

### PR DESCRIPTION
**Pull request by jkodu**
# Checklist for Pull Request
🚨Please review the [guidelines for contributing](https://auto-wcag.github.io/auto-wcag/pages/contribute.html) to this repository.

- [x] Make sure you are requesting to **pull a issue/feature/bugfix branch** (right side). Don't request your master branch!
- [x] Make sure you are making a pull request against the **master branch** (left side). Also you should start *your branch* off *master branch*.
- [x] Kindly prefix the pull request name with `DRAFT` or `FINAL` accordingly. For more details, read [Rule Design Guidelines](https://auto-wcag.github.io/auto-wcag/pages/design/review.html)
- [x] Provide a reference to the issue being addressed.

#Reviewing Guidelines
- For `DRAFT` PR's, leave any comments you have and approve if you’ve reviewed the entire pull request. 
- Once there have been three such reviews the pull request can be merged. 
- Any comments left will be taken up in an updated pull request, which will be necessary before a FINAL release”.
- For `FINAL` PR's, review and approve changes to be merged.

# How to Review And Approve
- Go to the “files changed” tab, there you will have the option to leave comments on different lines. 
- Once the review is completed, find the “Review changes” button in the top right, select “approve” and click “Submit review”

### Description
Final PR based on Draft - https://github.com/auto-wcag/auto-wcag/pull/146

💔Thank you!

### CODE REVIEW

**Comment by WilcoFiers** [comment]
I think this PR is good, but I want to see this "visible text content" definition before I give the OK on this.

**Comment by jkodu** [comment]
@jkodu - taking over

File: pages/algorithms/visible-text-content.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Visible Text Content.
+key: visible-text-content
+---
+
+Elements that are visible on the page, that are of type text, excluding text content in `title` or `alt` attributes, and are not categorised as [non text content](#https://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv-all.html). These elements should also be ensured to meet the color contrast and visibility requirements.
```
[the remark this answers is not in the document]
Comment (Brynanders) [reply pages/algorithms/visible-text-content.md]: Agreed andupdated

**Comment by Brynanders** [comment]
@carlosapaduarte 
'Images of text' are not text and would be considered 'visible non-text content'.
Good points about white-space and case sensitivity and the grammar spot :) I have updated the PR to reflect your requested changes.

**Comment by Brynanders** [comment]
@carlosapaduarte thanks for your comments, please review the changes and approve.
@WilcoFiers I have updated the applicability to include semantic role and removed the "1" from "Expectations"

**Comment by jkodu** [comment]
@cpandhi @jeankaplansky @skotkjerra - please review/ approve.

**Pull request merged**